src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -66,7 +66,41 @@ Cross identity ID copying is only allowed if the destination path has the the ri
 
 </Callout>
 
-### More `copy` options
+Alternatively, specify a non-default bucket target by including `bucket` in both the source and destination objects. 
```

Comment:
i would this as section - something like `Copy across bucket` with two different bucket names